Engine/DataFeeds/SubscriptionSynchronizer.cs

```diff
@@ -63,17 +62,17 @@ public SubscriptionSynchronizer(UniverseSelection universeSelection, DateTimeZon
         /// <param name="subscriptions">The subscriptions to sync</param>
         public TimeSlice Sync(IEnumerable<Subscription> subscriptions)
         {
-            long earlyBirdTicks;
             var changes = SecurityChanges.None;
             var data = new List<DataFeedPacket>();
             // NOTE: Tight coupling in UniverseSelection.ApplyUniverseSelection
             var universeData = new Dictionary<Universe, BaseDataCollection>();
             var universeDataForTimeSliceCreate = new Dictionary<Universe, BaseDataCollection>();
 
+            var frontierUtc = _timeProvider.GetUtcNow();
```

Comment:
awesome, call it once per invocation -- it'll still be slower since we're adding an extra loop, but this is great for now

Reply:
exactly, it will cause a second loop operation on the subscriptions, but since both loops are almost at the same moment of time it shouldn't cause an extra sorting of the subscriptions